packages/lib/getEventTypeById.ts

```diff
@@ -228,13 +228,17 @@ export default async function getEventTypeById({
       userId,
       app: {
         enabled: true,
+        categories: {
+          hasSome: [AppCategories.conferencing, AppCategories.video],
+        },
```

Comment:
In this function, we only get installed conferencing apps, so filter for those credentials only.

Reply:
On line 259, we are filtering on `credentials` for stripe payment which isn't a conferencing app, so that might break.